lib/pbench/server/api/resources/users_api.py

```diff
@@ -11,6 +11,57 @@
 from pbench.server.api.auth import Auth
 
 
+class PasswordRecovery(Resource):
+    def __init__(self, logger, auth, mail):
+        self.logger = logger
+        self.auth = auth
+        self.mail = mail
+
+    def post(self):
+        """
+        Post request for initiating password recovery.
+        This requires a JSON data with required user fields
+        {
+            "email": "user@domain.com"
+        }
+
+        Required headers include
+
+            Content-Type:   application/json
+            Accept:         application/json
+
+        :return:
+            Success: 201 with password reset link sent to email
+            Failure: <status_Code>,
+                    response_object = {
+                        "message": "failure message"
+                    }
```

Comment:
It looks like this function actually returns `OK`/`200` on success, not `201` (`CREATED`).

Also, I'm not sure that documenting the failure like that is particularly helpful.  (That's _really_ generic, and in fact the code actually calls the Flask `abort()` function on failure, so if someone wants to know the details they should presumably consult the documentation for _that_.)